packages/contracts/test/framework/dao/dao-registry.ts

```diff
@@ -168,85 +172,87 @@ describe('DAORegistry', function () {
   });
 
   // without mocking we have to repeat the tests here to make sure the validation is correct
-  describe('subdomain validation', () => {
-    it('should validate the passed subdomain correctly (< 32 bytes long subdomain)', async () => {
-      const baseSubdomain = 'this-is-my-super-valid-subdomain';
-
-      // loop through the ascii table
-      for (let i = 0; i < 127; i++) {
-        const newTargetDao = await deployNewDAO(signers[0]);
-
-        // replace the 10th char in the baseSubdomain
-        const subdomainName =
-          baseSubdomain.substring(0, 10) +
-          String.fromCharCode(i) +
-          baseSubdomain.substring(10 + 1);
-
-        // test success if it is a valid char [0-9a-z\-]
-        if ((i > 47 && i < 58) || (i > 96 && i < 123) || i === 45) {
+  skipTestSuiteIfNetworkIsZkSync('subdomain validation', async () => {
```

Comment:
You don't need the describe block after this

Reply:
https://github.com/aragon/osx/pull/596/commits/6fb8ca90e2f7185cad5d3d2909dfe5ef6357007a